### ISSUE TITLE
setup.sh does not download necessary libraries for 3.0

### ISSUE THREAD

**Issue by pmodernme** [firebase/FirebaseUI-iOS#22]
@mcdonamp
When I run setup.sh for the new 3.0 version, I get the following error (this is after it successfully downloads the firebase SDK):
```
-----------------------------------------------
STARTING FACEBOOK
-----------------------------------------------
Downloading Facebook SDK...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 55838    0 55838    0     0  41537      0 --:--:--  0:00:01 --:--:--  117k
Extracting Facebook SDK...
Archive:  ******/FirebaseUI-iOS/sdk/facebook-sdk.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of ******/FirebaseUI-iOS/sdk/facebook-sdk.zip or
        ******/FirebaseUI-iOS/sdk/facebook-sdk.zip.zip, and cannot find ******/FirebaseUI-iOS/sdk/facebook-sdk.zip.ZIP, period.
```

I put the ****** in the paths for privacy.
I have some new major updates that I would like to include, but I can't until I can get this to run properly.

**Comment by mcdonamp** [comment]
Interesting--looks like that version of the Facebook SDK no longer exists, so I switched the download link to https://origincache.facebook.com/developers/resources/?id=facebook-ios-sdk-current.zip which looks like it's pinned at the current version. We should likely do the same with Google as well.

**Comment by mcdonamp** [comment]
This is fixed in #24

**Comment by pmodernme** [comment]
Nice
